pkg/core/resource/resource.go

```diff
@@ -15,7 +15,7 @@ const pathPrefix = "/api/core/v1/resource.json"
 
 type (
 	Type = v1.Type
-	Info = v1.Info
+	Info = v1.Resource
```

Comment:
I assume you don't want to refactor the legacy client on purpose?

Reply:
exactly - no need to do a breaking change to something that is already deprecated anyway